.github/workflows/pdfbygger-brevbaker.yml

```diff
@@ -135,6 +135,7 @@ jobs:
           filters: |
             latex:
               - 'pdf-bygger/containerFiles/latex/**'
+              - 'pdf-bygger/latex.Dockerfile'
```

Comment:
Gir det mening å ha denne her? Det gir kanskje mer mening å basere det på pdf-bygger sin dockerfile, siden man må endre i den for å ta i bruk endringer i latex.Dockerfile?

Reply:
Tanken her var at endringar i latex.Dockerfile kan gjera at latex oppfører seg annleis, og at vi derfor bør køyre visuelle testar her - men tenkte ikkje da på den biten med at vi jo aktivt må ta i bruk nye endringar. Så ja, vi oppnår fort det eg ville oppnå ved å heller lytte på Dockerfile.

Kan gi ei og anna unødig køyring, men det er jo ikkje _så_ ofte vi endrar her, så det går nok fint.